### DISCUSSION TITLE
mesh inset

### DISCUSSION THREAD

**Discussion by darco415** [Jyers/Marlin#1139] [Q&A] [answered]
I compile my own fw. If I were to change my UBL bed mesh probe from 10x10 to 5x5, do I need to also adjust the mesh_inset? Currently it reads 10...I know to change the grid max points to 5..

 #define MESH_INSET 10          
 #define GRID_MAX_POINTS_X 10

thxs

**Comment by Jyers** [comment, marked as answer]
No, no real need to change that. It depends on how close the probe can get to the edge of the bed, ie to avoid clips and such.

**Reply by darco415** [reply]
ok cool, I dont use clips so Ill try to set at 0 and go from there,,,thxs
